cmd/serve-config.go

```diff
@@ -97,6 +107,15 @@ func serveConfigRun(cmd *cobra.Command, args []string) error {
 	awsSession, err := session.NewSessionWithOptions(
 		session.Options{
 			SharedConfigState: session.SharedConfigEnable,
+			Config: aws.Config{
+				Retryer: &client.DefaultRetryer{
```

Comment:
Unsure if this means we don't have to configure a retryer for IAM sessions

Reply:
This feels like the right place but I also have the same doubt as you. Not sure